proposals/lip-define_new_transaction_schema.md

```diff
@@ -177,13 +177,13 @@ def decodeTransaction(trsMsg: bytes) -> Transaction:
 
 ### Transaction Signature Calculation
 
-The following function calculates the signature with secret key `sk` of a transaction `trs` for a chain identified by the given [network identifier](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#network-identifiers).
+The following function calculates the signature with secret key `sk` of a transaction `trs` for a chain identified by the given [chainID](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md).
```

Comment:
I would still link to the section https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#chain-identifiers

Reply:
Indeed, I did not do it because the updated LIP37 was not merged yet..

Fixed